src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -246,6 +255,9 @@ TODO: Make this test work without requestVideoFrameCallback support (in waitForN
     const videoUrl = getResourcePath('red-green.webmvp8.webm');
     const videoElement = document.createElement('video');
     videoElement.src = videoUrl;
+    if (videoElement.canPlayType('video/webm; codecs=vp8') === '') {
```

Comment:
nit: I think we should put all of the mime types (and color spaces) for these files in a common place so reviewers don't have to verify them every time we duplicate them. How about an object that maps filename -> `{ mimeType, colorSpace }` similar to `kTextureFormatInfo` etc?

It can just live in this file (`video.spec.ts`) until some future point we need to move it.